src/material/expansion/expansion-panel.ts

```diff
@@ -204,6 +205,19 @@ export class MatExpansionPanel
           this._portal = new TemplatePortal(this._lazyContent._template, this._viewContainerRef);
         });
     }
+
+    this._ngZone.runOutsideAngular(() => {
+      if (this._animationsDisabled) {
+        this.opened.subscribe(() => this.afterExpand.emit());
+        this.closed.subscribe(() => this.afterCollapse.emit());
```

Comment:
This would be fired outside zone, which breaks zonejs-based apps listening to these outputs.